engine/runtime/src/main/java/org/enso/interpreter/runtime/builtin/Builtins.java

```diff
@@ -161,8 +161,12 @@ public Builtins(EnsoContext context) {
     duration = builtins.get(org.enso.interpreter.node.expression.builtin.date.Duration.class);
     timeOfDay = builtins.get(org.enso.interpreter.node.expression.builtin.date.TimeOfDay.class);
     timeZone = builtins.get(org.enso.interpreter.node.expression.builtin.date.TimeZone.class);
-    special = new Special(language);
     warning = builtins.get(Warning.class);
+
+    error = new Error(this, context);
```

Comment:
Does this modification has any meaning? Is the order of the initialization of the builtins now important?

Reply:
It was when I was trying to read `Nothing` from `Error` constructor. First of all I am initializing the _no code_ `builtins.get` and only then I am initializing objects with complex constructors that can make callbacks to `Builtins` again.

Not really necessary anymore, but probably more sane than the original version.